web/docs/introduction/what-is-wasp.md

```diff
@@ -0,0 +1,174 @@
+---
+title: What is Wasp?
+---
+
+import ImgWithCaption from '../../blog/components/ImgWithCaption'
+
+We will give a brief overview of what Wasp is, how it works on a high level and when to use it.
+
+## Wasp is a tool to build modern web applications
+
+It is an opinionated way of building **full-stack web applications**. It takes care of all three
+major parts of a web application: **client** (front-end), **server** (back-end) and **deployment**.
+
+#### Works well with your existing stack
+Wasp is not trying to do everything at once but rather focuses on the complexity
+which arises from connecting all the parts of the stack (client, server, deployment) together.
+
+Wasp is using **React**, **Node.js** and **Prisma** under the hood and relies on them to define web components and server queries and actions.
+
+#### Wasp's secret sauce
+
+At the core is the Wasp compiler which takes the Wasp config and your Javascript code and outputs the client app, server app and deployment code.
+
+<!-- ![Wasp compilation diagram](/img/lp/wasp-compilation-diagram.png) -->
+
+<ImgWithCaption
+    source="/img/lp/wasp-compilation-diagram.png"
+    caption="How the magic happens 🌈"
+/>
+
+The cool thing about having a compiler that understands your code is that it can do a lot of things for you. 
+
+Define your app in the Wasp config and get:
+- login and signup with Auth UI components,
+- full-stack type safety,
+- e-mail sending,
+- async processing jobs,
+- React Query powered data fetching,
+- security best practices,
+- and more. 
+
+You don't need to write any code for these features, Wasp will take care of it for you 🤯
```

Comment:
I would maybe also consider adding that we continue to update/improve/maintain the "Wasp-owned code" over time, not just spit something out that _you_ have to then maintain after initial project generation. (aka a counterpunch at T3 and similar)

Reply:
That's a good idea, I'll add it 👍